Assets/Mirror/Runtime/NetworkReaderPool.cs

```diff
@@ -1,41 +1,63 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 
 namespace Mirror
 {
-    public static class NetworkReaderPool
+    public class NetworkReaderPool : NetworkReader, IDisposable
```

Comment:
using NetworkReaderPool reader = ...
sounds like 'reader' is a pool, even though it's a pooled reader.
NetworkReaderPool**ed** would be a more obvious name

Reply:
Renaming can come in a separate PR

Reply:
please rename in this PR. no need to add unnecessary breaking changes

Reply:
Merged changes from #1492